src/main/java/dk/dbc/updateservice/actions/UpdateOperationAction.java

```diff
@@ -30,10 +30,8 @@
 import java.time.format.DateTimeFormatter;
 import java.time.format.DateTimeFormatterBuilder;
 import java.time.temporal.ChronoField;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Properties;
-import java.util.Set;
+import java.util.*;
```

Comment:
Er stjerne imports accepteret af is'en ?

Reply:
Yep, det er std.